### PR TITLE
feat: Add Null values at the last while sorting

### PR DESCRIPTION
**Pull request by tejaswinichile**
## Description

Consider null values at the last while applying sort

### CODE REVIEW

**Comment by tejaswinichile** [comment]
@sojan-official Added the spec for checking the null values remains at last.

**Comment by pranavrajs** [comment]
@tejaswinichile Made a small change to change the sort on county using the name instead of code. The reason is that the country for some countries is different from its name (for eg: United Arab Emirates is AE). This would come before Argentina and would create confusion that the sort is not working.

File: app/javascript/dashboard/routes/dashboard/contacts/components/ContactsTable.vue [hunk 1]
```diff
@@ -178,12 +178,14 @@ export default {
         {
           field: 'company',
           key: 'company',
+          sortBy: this.sortConfig.company_name || '',
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/contacts/components/ContactsTable.vue:181]: Instead of `''` we could use `undefined`

**Review by nithindavid — CHANGES_REQUESTED** [review]

**Comment by tejaswinichile** [comment]
@pranavrajs @sojan-official Please have a look at this one.

**Review by pranavrajs — APPROVED** [review]

**Review by sojan-official — APPROVED** [review]

File: app/javascript/dashboard/routes/dashboard/contacts/components/ContactsTable.vue (hunk 1 above)
Comment (tejaswinichile) [reply app/javascript/dashboard/routes/dashboard/contacts/components/ContactsTable.vue:181]: @nithindavid I have used it as we used it for other filters, e.g email, and phone_number. Should I change from `''` to `undefined` for the previous filters too?

**Review by nithindavid — APPROVED** [review]

**Pull request merged**